### PR TITLE
Create universal_asset_id.md

### PR DESCRIPTION
**Pull request by vikmeup**

### CODE REVIEW

File: assets/universal_asset_id.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+- Bitcoin: `c0`
+- Ethereum: `c60`
+- Binance Chain `c714`
+
+### Tokens:
+
+- DAI (Ethereum): `c60_t0x6B175474E89094C44Da98b954EedeAC495271d0F`
+- BUSD (Binance Chain): `c714_tBUSD-BD1`
+- USDT (Tron): `c195_tTR7NHqjeKQxGTCi8q8ZY4pL8otSzgjLj6t`
```
Comment (EnoRage) [inline assets/universal_asset_id.md:22]: Invariant? 
`c60_t0x6B175474E89094C44Da98b954EedeAC495271d0F` and `t0x6B175474E89094C44Da98b954EedeAC495271d0F_c60` ?

File: assets/universal_asset_id.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+### Coins:
+
+- Bitcoin: `c0`
+- Ethereum: `c60`
+- Binance Chain `c714`
+
+### Tokens:
+
+- DAI (Ethereum): `c60_t0x6B175474E89094C44Da98b954EedeAC495271d0F`
+- BUSD (Binance Chain): `c714_tBUSD-BD1`
```
Comment (hewigovens) [inline assets/universal_asset_id.md:21]: how to express smart chain bnb?

Comment (leoneparise) [reply assets/universal_asset_id.md:21]: How different is Smart chain BNB?

File: assets/universal_asset_id.md (hunk 1 above)
Comment (vikmeup) [reply assets/universal_asset_id.md:22]: Let's only use one way for simplicity

File: assets/universal_asset_id.md (hunk 2 above)
Comment (vikmeup) [reply assets/universal_asset_id.md:21]: Added details on coinId, which kinda sucks because there is a divergence from slip44

**Pull request merged**